Dockerfile_amd

```diff
@@ -104,7 +104,7 @@ RUN case ${TARGETPLATFORM} in \
     /opt/conda/bin/conda clean -ya
 
 # Install flash-attention, torch dependencies
-RUN python3 -m pip install --upgrade pip && pip install numpy einops ninja joblib msgpack cmake --no-cache-dir && rm -rf /var/lib/apt/lists/*
+RUN python3 -m pip install --upgrade pip uv && pip install numpy einops ninja joblib msgpack cmake --no-cache-dir && rm -rf /var/lib/apt/lists/*
```

Comment:
```suggestion
RUN python3 -m pip install --upgrade pip uv && uv pip install numpy einops ninja joblib msgpack cmake --no-cache-dir && rm -rf /var/lib/apt/lists/*
```

p.s. besides the comment, shouldn't `rm -rf /var/lib/apt/lists/*` be placed after an `apt-get` somewhere for consistency, instead of after the `pip install`?

Reply:
It's probably bad copy-paste. I'm not touching anything non essential. This is a significant PR already